### PR TITLE
Upgrade to 3.1.8 and auto-add share user

### PR DESCRIPTION
**Pull request by rrva**
- Supports setting environment variables for auto-adding a single user/password.
  
  AFP_USER       user to create and allow to access the share
  AFP_PASSWORD   password
  AFP_UID        uid of created user
  AFP_GID        gid of created user
- Upgraded base image to debian jessie
- Upgraded netatalk to 3.1.8
- Replaced supervisord with start.sh which does tail -f of netatalk.log
- Reworked dependency list and netatalk build according to [Netatalk wiki: Install Netatalk 3.1.8 on Debian 8 Jessie](http://netatalk.sourceforge.net/wiki/index.php/Install_Netatalk_3.1.8_on_Debian_8_Jessie)
- Trimmed 46Mb from image, so now 201.4Mb.
- ~~LDAP libraries removed~~ (edit: LDAP untested but was included)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,18 +46,36 @@ If you would like to use an editor like nano, emacs, or vi you have to install i
...
 
 ```ini
-[Secured Share]
-path = /media/secure
-valid users = timemachine
+[Global]
+log file = /var/log/netatalk.log
+
+[share]
+path = /share
```
Comment (cptactionhank) [inline README.md:77]: Why change away from `/media` or why not use a directory like `/media/share`. This kinda matches the [FHS 3.0](http://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#purposeMediaMountPoint) as `/media` contains mount points for removable media which it kinda will be with the described way of mounting an external directory into the container.

**Comment by cptactionhank** [comment]
This is sincerely awesome, thank you for doing this. Could you have a look at the comment to the commit I made.

File: afp.conf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[Global]
+log file = /var/log/netatalk.log
+
+[share]
+    path = /share
```
Comment (cptactionhank) [inline afp.conf:5]: COPY: Why change away from `/media` or why not use a directory like `/media/share`. This kinda matches the FHS 3.0 as /media contains mount points for removable media which it kinda will be with the described way of mounting an external directory into the container.

File: README.md (hunk 1 above)
Comment (rrva) [inline README.md:77]: Sure, this can be changed, I thought /share was shorter to type and more generic as default but sure.

**Comment by rrva** [comment]
@cptactionhank: No problem with changing back to /media/share, I thought the path was a bit long and it sounded like you had "media files" there.

Some small points I thought of
- I might have broken LDAP authentication, untested
- There are some "rm -rf" steps at the end of the Dockerfile which are probably unneccessary

**Comment by cptactionhank** [comment]
Thank you for this, a newly edited README and path changes is on its way

**Comment by rrva** [comment]
@cptactionhank: Mention the mount_afp command in README while you're at it:

Something like:

Examples:

To mount a share from docker-machine, run the following on your OSX host

``` bash
mkdir $HOME/share
mount_afp afp://<user>:<password>@$(docker-machine ip)/share $HOME/share
```

**Pull request merged**